### PR TITLE
Fixes reagent scanner not showing time of death correctly

### PR DESCRIPTION
**Pull request by Robertha89**
Bugfix, can be merged at any time.

### CODE REVIEW

File: code/game/objects/items/devices/scanners.dm [hunk 1]
```diff
@@ -124,6 +124,7 @@ REAGENT SCANNER
 	origin_tech = "magnets=1;biotech=1"
 	var/upgraded = 0
 	var/mode = 1;
+	var/timeofdeath = null
```
Comment (tigercat2000) [inline code/game/objects/items/devices/scanners.dm]: I can't even begin to articulate what is wrong about this.

This isn't showing their time of fucking death, this is asking for the worldtime2text of a null value.

Comment (Robertha89) [inline code/game/objects/items/devices/scanners.dm]: oh, herp de fucking derp.

**Comment by Robertha89** [comment]
Okay, it's fixed now.

**Comment by Robertha89** [comment]
Instead of showing timeofdeath as a clock, it showed it as an amount of seconds, which generaly explained nothing, but now it works.

**Pull request merged**